portfolio/src/main/java/com/google/sps/utils/ServletUtils.java

```diff
@@ -18,4 +18,27 @@ public static String getParameter(HttpServletRequest request, String name, Strin
         }
         return value;
     }
+
+    /**
+     * Gets the integer parameter from html form input.
+     * @param request Http request.
+     * @param parameterName Input id in html element.
+     * @param defaultValue Default integer.
+     * @return Parsed integer or default value if exception occur
+     */
+    public static int getIntParameter(HttpServletRequest request, String parameterName, int defaultValue) {
+        String maxNumOfCommentStr = request.getParameter(parameterName);
```

Comment:
nit, naming is not correct it should be generic since you are in a util class.

Reply:
Fixed